README.md

```diff
@@ -40,6 +40,7 @@ If you have a question or an idea, you can create an issue. Pull requests are we
 
 ## Authors
 - **Nicolas Ettlin** ([@Nicolapps](https://github.com/Nicolapps))
+- **Adam Mathieson** ([@AMathieson](https://github.com/amathieson)) - Help in transition to alternative API & Updates to vehicles
```

Comment:
Thank you for the change, you definitely deserve credit with all the help you’ve been providing!